ykman-gui/qml/PivPinPromptDialog.qml

```diff
@@ -0,0 +1,63 @@
+import QtQuick 2.5
+import QtQuick.Dialogs 1.2
+import QtQuick.Controls 1.4
+import QtQuick.Layouts 1.1
+
+Dialog {
+
+    property var callback
+    property var defaultValue
+    property string errorMessage: ''
+    property bool hideInput: true
+    property string message: qsTr('Please enter the PIV PIN.')
```

Comment:
Let's just call it PIN.

Reply:
What if it's different from the OpenPGP or FIDO2 PIN?